sim/src/main/scala/spinal/sim/XSimBackend.scala

```diff
@@ -111,6 +124,15 @@ class XSimBackend(config: XSimBackendConfig) extends Backend {
       throw new Exception(message)
     }
   }
+  def doCmdMys2(command: String, cwd: File, message : String) = {
+    val cmd_msys2 = s""" $msys2Shell -defterm -here -no-start -mingw64 -c "$command" """.trim
```

Comment:
this command would not use env variable in msys if you start it through cmd shell on Windows.

Reply:
yes, because on windows the build of `spinal_xsim.dll` depends only on the `VIVADO_HOME` env var to copy locally needed include files.

This behavior was already there before those commits, and I kept it,
as it allows to have this only step which actually doesn't depends on any PATH (cmd or msys) env var.